quickstart/latest/Fiddler_Quickstart_Imbalanced_Data.ipynb

```diff
@@ -55,7 +54,19 @@
     "id": "hcP0yWfV1GoZ"
    },
    "source": [
-    "## 1. Connect to Fiddler"
+    "# 1. Connect to Fiddler\n",
+    "\n",
+    "Before you can add information about your model with Fiddler, you'll need to connect using our API client.\n",
```

Comment:
Do we call it API client?  Seems inconsistent to me as I typically see it called Fiddler Python client.